### ISSUE TITLE
Failed Build when replacing a custom module with same module now in contrib via composer

### ISSUE THREAD

**Issue by mrdavidburns** [Lullabot/drainpipe#178]
https://github.com/Lullabot/drainpipe/blob/d97e778fb965e98dc4026811524e1ec473e0b0d9/tasks/drupal.yml#L56

Ran into this error when replacing a custom module that is now available in contrib.
```
+ ./vendor/bin/task drupal:update
[drupal:update]  [error]  AssertionError: The file specified by the given app root, relative path and file name (/var/lib/tugboat/web/modules/custom/entity_browser_block_layout/entity_browser_block_layout.info.yml) do not exist. in assert() (line 68 of /var/lib/tugboat/web/core/lib/Drupal/Core/Extension/Extension.php) #0 /var/lib/tugboat/web/core/lib/Drupal/Core/Extension/Extension.php(68): assert(false, 'The file specif...')
[drupal:update] #1 /var/lib/tugboat/web/core/lib/Drupal/Core/Extension/ModuleHandler.php(114): Drupal\Core\Extension\Extension->__construct('/var/lib/tugboa...', 'module', 'modules/custom/...', 'entity_browser_...')
[drupal:update] #2 /var/lib/tugboat/web/core/lib/Drupal/Component/DependencyInjection/Container.php(259): Drupal\Core\Extension\ModuleHandler->__construct('/var/lib/tugboa...', Array, Object(Drupal\Core\Cache\DatabaseBackend))
```

**Comment by deviantintegral** [comment]
@mrdavidburns can you run this with `--verbose` so we can see which drush command is triggering that? I'm guessing it's the plugin cache clear, but it would be good to know.

**Comment by dsayswhat** [comment]
It's failing on the first action in drupal:update, which is `./vendor/bin/drush {{.site}} --yes cache:clear plugin`.

I've verified in the tugboat terminal that calling `drush cache:clear plugin` crashes with the same error.

The particular thing we're seeing when this started occuring is that we deleted a custom module and replaced it with a contrib module that does the same thing... see https://github.com/Lullabot/lullabot.com-d8/pull/2527. 

Dave and I have discussed the possibility that we need to uninstall the module before trying to delete the files, but it seems to work on his local. 

Given all that, I don't know whether this is Drainpipe or an order-of-operations problem in our attempt to uninstall the module... any thoughts are appreciated.
